check.py

```
@@ -1,3 +1,7 @@
+# def login(username, password):
+#     query = f"SELECT * FROM users WHERE username='{username}' AND password='{password}'"
+#     return db.execute(query).fetchone()
+
 def login(username, password):
     query = f"SELECT * FROM users WHERE username='{username}' AND password='{password}'"
     return db.execute(query).fetchone()
```

Ruff (0.12.2)

Undefined name `db`

(F821)